.travis.yml

```diff
@@ -2,13 +2,13 @@ language: node_js
 node_js:
  - '10'
  - '8'
- - '6'
```

Comment:
You shouldn't really remove this if you are still officially supporting WP4.

Reply:
We still officially support webpack 2 and 3 as well; following your logic we should add node 4 back into the test matrix? 😄 

In all seriousness, support is always something we take seriously.  However, given [node 6 is already in "Maintenance" and is being end of lifed in April](https://nodejs.org/en/blog/release/v6.9.0/) there seems little merit in continuing to support it.  No-one should be actively using node 6 at this stage.

Reply:
Well, why not transpile to ES2015 then? ^_^

Reply:
We do: https://github.com/Realytics/fork-ts-checker-webpack-plugin/blob/51981e79106a771b5f05eddec42c06eb13529523/src/tsconfig.json#L3

Reply:
This is why I'm confused. You know that's not compatible with node 4, right? How can you say you are compatible with WP2? Probably no one is using in a node 4 environment and that's ok, but that seems to be just a happy accident. O_o

Reply:
No. 

This plugin never explicitly targeted node 4 AFAIK, looking at the history of the project. 

The misconception is that node `^4.0.0` is _absolutely_ required to run this plugin alongside webpack2, which is false. **Users can freely choose which ever node versions they want to run on, as long as all the dependencies' runtime requirements are fulfilled.**

We've only recently changed to target es6 instead of es5. [Node 4 already supports 97% of es5 features](http://kangax.github.io/compat-table/es5/#node4), which was why it worked most of the time.

With the introduction of webpack5 support (with backwards compatibility of older webpack versions), we needed to bump the minimum required node version to `>=8.9.0` because webpack5 requires it.

We're talking mostly about language features; compatibility with webpack2 doesn't mean that it has to 100% run on node4, rather webpack2 itself requires at least node 4. So you could go off running webpack 2 on Node 11 with the latest version of this plugin with no issues ([source](https://github.com/webpack/webpack/blob/webpack-2/package.json#L72)).